llvm/test/Transforms/InstCombine/fold-xor-and-select-i1.ll

```diff
@@ -0,0 +1,14 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py
```

Comment:
Move your tests into `llvm/test/Transforms/InstCombine/select-and-or.ll`, instead of creating new files.